rwgame/debugstate.cpp

```diff
@@ -147,6 +147,7 @@ DebugState::DebugState(RWGame* game, const glm::vec3& vp, const glm::quat& vd)
 
 void DebugState::enter()
 {
+	getWindow().hideCursor();
```

Comment:
actually I think we'd like to show the mouse cursor in DebugState unless the free flight Camera is in use?
[should be a function somewhere else: make the mouse cursor invisible and center it OR show the cursor but release it]